core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
@JDutil All instances of `Spree::Numbergenerator` now implement `#prefix`, `#prefix=`, `#length`, `#length=`.

Also each class that includes an instance of `Spree::NumberGenerator` gets an `.number_generator` accessor. So calls like `Spree::Order.number_generator.prefix = 'FOO'` will work now.